backend/de.metas.business/src/main/java/de/metas/order/model/interceptor/C_Order.java

```diff
@@ -230,17 +230,20 @@ public void setDeliveryViaRule(final I_C_Order order)
 	@CalloutMethod(columnNames = I_C_Order.COLUMNNAME_C_BPartner_ID)
 	public void setPaymentRule(final I_C_Order order)
 	{
-		final I_C_BPartner bpartner = order.getC_BPartner();
-		final PaymentRule paymentRule;
-		if (bpartner != null && bpartner.getPaymentRule() != null)
-		{
-			paymentRule = PaymentRule.ofCode(bpartner.getPaymentRule());
-		}
-		else
+		if (!InterfaceWrapperHelper.isCopying(order))
```

Comment:
please add javadoc to explain why this method only acts `if (!InterfaceWrapperHelper.isCopying(order))`